From 712eeaa2e042f8f4f36d04afb2727852a0741075 Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Tue, 15 Mar 2022 18:41:46 +0100
Subject: [PATCH 0240/1141] dt-bindings: remoteproc: revert compatibility for
 TEE support

The "st,stm32mp1-m4_optee" compatible value is no more used in the
rproc_stm32.c driver.

This reverts commit 7db4f42a06f8ab316479016b47c61c5cd56d16e9.

Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: I44c3b5709e8828d26a48144101baaca9b0634167
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/244281
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 .../devicetree/bindings/remoteproc/st,stm32-rproc.yaml        | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

--- a/Documentation/devicetree/bindings/remoteproc/st,stm32-rproc.yaml
+++ b/Documentation/devicetree/bindings/remoteproc/st,stm32-rproc.yaml
@@ -16,9 +16,7 @@ maintainers:
 
 properties:
   compatible:
-    enum:
-      - st,stm32mp1-m4
-      - st,stm32mp1-m4_optee
+    const: st,stm32mp1-m4
 
   reg:
     description:
